trimmer_decorator.rb

```diff
@@ -0,0 +1,12 @@
+require './base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
```

Comment:
- [ ] Please the correct_name method should converte  check the name length if it's greater than 10 it should return
a slice of the name
`     @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name`